includes.php

```diff
@@ -276,20 +276,10 @@ function format_linked_tasks( array $linkedTasks ): string {
 	return $linkedTasks ?: '<em>No tasks</em>';
 }
 
-function make_shell_command( array $env, string $cmd ): string {
-	$prefix = '';
-	foreach ( $env as $key => $value ) {
-		$value = escapeshellarg( $value );
-		$prefix .= "$key=$value ";
-	}
-
-	return "$prefix$cmd 2>&1";
```

Comment:
To be clear: the `2>&1` is also not needed, because the `run( … )` callback will echo both stdout and stderr.